cloud_info_provider/tests/test_static.py

```diff
@@ -167,21 +167,60 @@ def test_get_template_defaults(self):
         yaml = {'compute': {'templates': {'defaults': {'foo': 'bar',
                                                        'baz': 'bazonk'}}}}
         self.provider.yaml = yaml
-        self.assertEqual({'foo': 'bar', 'baz': 'bazonk'},
+        self.assertEqual({'foo': 'bar', 'baz': 'bazonk', 'network_out': True},
                          self.provider.get_template_defaults())
-        self.assertEqual({'template_foo': 'bar', 'template_baz': 'bazonk'},
+        self.assertEqual({'template_foo': 'bar', 'template_baz': 'bazonk',
+                          'template_network_out': True},
                          self.provider.get_template_defaults(prefix=True))
 
     def test_get_compute_endpoint_defaults(self):
         yaml = {'compute': {'endpoints': {'defaults': {'foo': 'bar',
                                                        'baz': 'bazonk'}}}}
         self.provider.yaml = yaml
-        self.assertEqual({'foo': 'bar', 'baz': 'bazonk'},
-                         self.provider.get_compute_endpoint_defaults())
-        self.assertEqual(
-            {'compute_foo': 'bar', 'compute_baz': 'bazonk'},
-            self.provider.get_compute_endpoint_defaults(prefix=True)
-        )
+        unprefixed = self.provider.get_compute_endpoint_defaults()
+        self.assertEqual(unprefixed.pop('foo'), 'bar')
+        self.assertEqual(unprefixed.pop('baz'), 'bazonk')
+        self.assertEqual({'api_authn_method': 'oidc',
+                          'api_endpoint_technology': 'webservice',
+                          'capabilities': [
+                              'executionmanagement.dynamicvmdeploy',
+                              'security.accounting'
+                          ],
+                          'failover': False,
+                          'live_migration': False,
+                          'max_dedicated_ram': 0,
+                          'min_dedicated_ram': 0,
+                          'production_level': 'production',
+                          'service_capabilities': [
+                              'executionmanagement.dynamicvmdeploy',
+                              'security.accounting'
+                          ],
+                          'service_production_level': 'production',
+                          'total_cores': 0,
+                          'total_ram': 0,
+                          'vm_backup_restore': False}, unprefixed)
+        prefixed = self.provider.get_compute_endpoint_defaults(prefix=True)
```

Comment:
It seems that here you are doing two different tests (unprefixed and prefixed) in the same test, so it could be possible to split this if you think it's worth.

Reply:
this is following how the rest of tests are coded I would not change it for the time being
Honestly I don't know why we have the prefix, probably is not needed anymore.

Reply:
OK, so maybe we should add an issue somewhere to rembember to check this and delete if no more useful?